packages/codegen-doc/src/fragment.ts

```diff
@@ -25,8 +25,24 @@ export function findFragment(
 /**
  * Check whether this object has valid content and is not a connection, edge, root or has a skip comment.
  */
-export function isValidObject(context: PluginContext, fragment: NamedFields<ObjectTypeDefinitionNode>): boolean {
+export function isValidObject(
+  context: PluginContext,
+  fragment: NamedFields<ObjectTypeDefinitionNode>,
+  previousFragments: Fragment[]
```

Comment:
If I understand correctly this is a one-pass and thus we rely here on the `ExternalUser` fragment appearing before `ExternalUserConnection` or we could discard something valid? Am I misunderstanding or do we have good reason to believe that the fragment we depend on will always show up first?

Reply:
Correct. This relies on the assumption that the base model, the one we have the ability to mark as skipped, will always appear before `BaseModelConnection` and `BaseModelEdge`. 
If this assumption breaks, we'll have to push this check further down the line, before printing the Fragment, which happens after the FragmentVisitor has completed his job and thus should have a complete view.

Let me check if it makes sense to do this change now.

Reply:
Actually, it's not that easy to check after all the fragments have been visited. The Fragment visitor prints fragments as strings right during visit. Changing this to wait until all fragments have been visited would require a major overhaul of the plugin.